Bio/Seq.py

```diff
@@ -1391,7 +1394,7 @@ def complement(self):
         return self
 
     def reverse_complement(self):
-        """The reverse complement of an unknown nucleotide equals itself.
+        """Return the reverse complement of an unknown nucleotide equals itself.
```

Comment:
The function itself does not make sense.

How about `Return the reverse complement of an unknown nucleotide,which equals itself.`?